.github/workflows/integration-tests.yml

```diff
@@ -27,23 +25,21 @@ jobs:
       - uses: jpribyl/action-docker-layer-caching@v0.1.1
         with:
           skip-save: true
-          continue-on-error: true
       - name: Run all up happy-path test
         run: tests/all-up-test.sh
         env:
           HARDHAT: True
-  relay-market:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: jpribyl/action-docker-layer-caching@v0.1.1
-        with:
-          skip-save: true
-          continue-on-error: true
-      - name: Run all up relay market test
-        run: tests/all-up-test.sh RELAY_MARKET $ALCHEMY_ID
-        env:
-          ALCHEMY_ID: ${{ secrets.ALCHEMY_ID }}
+  # relay-market:
```

Comment:
Why is it disabled?

Reply:
I disabled the relay market test because we almost always have to cancel it (I think the last time I saw it complete was 2 months ago). It uses a remote chain, I wonder if it is possible to do it all locally instead.